publish-aar.gradle

```diff
@@ -1,4 +1,4 @@
-//Version 2.1.1
+//Version 3.0.1
```

Comment:
Doesn't look like this file has changed - should be 2.2.2 from https://github.com/lampkicking/build-ci-tools/blob/master/publication/android/aar.gradle

Reply:
Ops...misunderstood that step 😅